jwst/model_blender/rules.py

```diff
@@ -0,0 +1,81 @@
+from datetime import time
+
+import numpy as np
+
+
+__all__ = ['RULE_FUNCTIONS', 'Blender', 'make_blender']
+
+
+def multi(vals):
+    """
+    This will either return the common value from a list of identical values
+    or 'MULTIPLE'
+    """
+    uniq_vals = list(set(vals))
+    num_vals = len(uniq_vals)
+    if num_vals == 0:
+        return None
+    if num_vals == 1:
+        return uniq_vals[0]
+    if num_vals > 1:
+        return "MULTIPLE"
```

Comment:
Could a unit test for the Blender class be added that touches the `"MULTIPLE"` return and ensures that everything works properly when the list isn't identical?

Reply:
Good idea! I added one in: d75874c48